### DISCUSSION TITLE
about implementation

### DISCUSSION THREAD

**Discussion by rraammiinn** [reflex-dev/reflex#572] [General]
does it uses pyscript and if not how it can translate python modules to js ?

**Comment by Alek99** [comment]
It doesn't we built it custom ourselves.

**Comment by Alek99** [comment]
The frontend compiles to a react app

**Reply by rraammiinn** [reply]
what about binary dependencies of python modules ?

**Comment by rraammiinn** [comment]
can I use any python module directly in frontend without connecting to backend ?
